dist/index.js

```diff
@@ -54392,7 +54392,22 @@ var __webpack_exports__ = {}
     Object.assign(this, { Compiler: compiler })
   } // CONCATENATED MODULE: ./node_modules/remark-stringify/index.js
 
-  /* harmony default export */ const remark_stringify = remarkStringify
+  /* harmony default export */ const remark_stringify = remarkStringify // CONCATENATED MODULE: ./node_modules/strip-final-newline/index.js
+
+  function stripFinalNewline(input) {
+    const LF = typeof input === 'string' ? '\n' : '\n'.charCodeAt()
+    const CR = typeof input === 'string' ? '\r' : '\r'.charCodeAt()
+
+    if (input[input.length - 1] === LF) {
+      input = input.slice(0, -1)
+    }
+
+    if (input[input.length - 1] === CR) {
+      input = input.slice(0, -1)
+    }
+
+    return input
+  }
```

Comment:
@PatrickHeneise : Are there situations where `trim()` wouldn't be ideal?

Trying to think through the change, we're effectively wanting to clip the whitespace off the ends.  So even if 'trim()' is too much, maybe 'rtrim()'?

Reply:
You're in `dist`, so this piece comes from https://www.npmjs.com/package/strip-final-newline:

> Normally, you would use stdout.trim(), but that would also remove newlines at the start and whitespace.